src/index.html

```diff
@@ -16,17 +16,25 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page_body">
+    <div class="search-bar">
+      <div class="search-bar__big search-bar__field"
+           data-qa="big">
+        <input
+        type="text"
+        data-qa="keypress"
+        placeholder="Try &#8220;Los Angeles&#8221;"
+      />
+      </div>
+      <div class="search-bar__small search-bar__field"
+          data-qa="small">
```

Comment:
Similar to the previous comment, this `<div>` element is not properly closed. Ensure it has a closing tag `</div>`.